e2e-tests/batch-mode-tests.sh

```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/env bash
 
-# TODO add
-# set -e
+set -e
```

Comment:
While you are at this please add the license text at the top too plus a short description of what this file is.

BTW, now that you have separated run and validation, how about renaming this file too, e.g., `batch-mode-validation.sh`?